### PR TITLE
Initial cleanup

### PR DESCRIPTION
**Pull request by Petermarcu**
This is cleaning up the upper part of the readme to update it and make it more concise and focused on what people will be expecting to see. Also fixing psjson6 links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,22 @@
...
+## Package List
 
-For a full list of packages available for download in this repository, please see our [list of .NET SDK packages](https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/Documentation/sdk-for-net-packages.md).
+For a full list of packages built from this repository, please see our [list of .NET SDK packages](https://github.com/Azure/azure-sdk-for-net/tree/master/Documentation/sdk-for-net-packages.md).
 
 ### Target Frameworks:
 
-* .NET Framework 4.5.2
-* Netstandard 1.4, based on the NetCore framework
+* .NET Framework 4.6.1
```
Comment (weshaggard) [inline README.md]: We need to be careful about changing this all up for the entire repo. @shahabhijeet when do you plan to move to these targets for management?

Comment (Petermarcu) [reply README.md]: @shahabhijeet how do you think we should message this to be clear that new libraries and major (possibly minor) rev's of the library should change their baseline and that this is the baseline moving forward?

Comment (Petermarcu) [reply README.md]: Note, I can remove these changes to get the rest of it through if thats better. We do need to figure this one out too though.

Comment (weshaggard) [reply README.md]: Yeah I'd suggest being careful about this part of your change as we need to understand the plan and then message it.

**Comment by Petermarcu** [comment]
Thanks for the review @weshaggard . Merging.

File: README.md [hunk 2]
```diff
@@ -77,7 +66,7 @@ dotnet test SDKs\Compute\Compute.Tests\Compute.Tests.csproj
 
 ### Project Structure
 
-In "SDKs\< RPName >", you will find projects for services that have already been implemented
+In "SDKs\< Service Name >", you will find projects for services that have already been implemented
```
Comment (shahabhijeet) [inline README.md]: @Petermarcu the last I checked with @weshaggard the new directory structure is not going to be until we are done with Swagger to SDK work.

Comment (Petermarcu) [reply README.md]: What is the difference between RP Name and Service name? Is RP a concept that we expect external contributors to understand? I was trying to use simpler words, not change to the new structure.

**Pull request merged**